co-log/tutorials/1-intro/Intro.md

```diff
@@ -43,22 +43,22 @@ example0 = do
 ```
 
 Using `putStrLn` for logging is a very simple and basic approach for logging.
-When your application becomes bigger and more complex, you might want to bring
-some logging library into it. For example, you might want to do something from
-the following list:
-
-1. Specify messages with the given `Severity` so you can control the verbosity
-   of the output.
-2. Automatically print timestamps, thread ids, source code line of the logging
-   with each message.
-3. You would like to submit some statistics to some web-server with each logging
-   message so later you can have analytics provided by external parties.
-
-Now let's look at how you can use `LogAction` instead of `putStrLn` to achieve the
-same goal. With `co-log` you need to have a value of type `LogAction` that defines
-how you are going to do logging. So you configure your logging settings separately and
-then pass and use this `LogAction` value. See the following example for more
-details:
+When your application grows bigger and more complex, you might want to introduce
+some logging library to it. For example, you might want to do something from the
+following list:
+
+1. Specify messages with a given `Severity` so you can control the verbosity of
+the output.
+2. Automatically print timestamps, thread IDs, and source code lines of the log
+messages.
```

Comment:
I guess what was meant is that each message will have the listed information, not this is the information *about* the message.